### PR TITLE
Add packet_volume_attachment resource

### PR DESCRIPTION
**Pull request by t0mk**
This PR adds resource for volume attachment in the Packet provider.

### CODE REVIEW

**Comment by t0mk** [comment]
This is ready for review.

**Comment by bzub** [comment]
LGTM

File: website/docs/r/volume_attachment.html.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Device and volume must be in the same location (facility).
+
+Once attached by Terraform, they must then be mounted using the `packet_block_attach` and `packet_block_detach` scripts.
+
+## Example Usage
+
+```hcl
+  resource "packet_project" "test_project" {
+      name = test-project"
```
Comment (bzub) [inline website/docs/r/volume_attachment.html.markdown]: missing quotation mark

Comment (t0mk) [reply website/docs/r/volume_attachment.html.markdown]: thanks. fixed

**Pull request merged**